src/controller/CommissioningWindowOpener.cpp

```diff
@@ -36,7 +36,8 @@ namespace chip {
 namespace Controller {
 
 CHIP_ERROR CommissioningWindowOpener::OpenBasicCommissioningWindow(NodeId deviceId, Seconds16 timeout,
-                                                                   Callback::Callback<OnOpenBasicCommissioningWindow> * callback)
+                                                                   Callback::Callback<OnOpenBasicCommissioningWindow> * callback,
+                                                                   bool jointCommissioning)
 {
```

Comment:
Spec, section 12.2.5: "A new RANDOM passcode SHALL be computed and OpenJointCommissioningWindow SHALL be
called using the corresponding PAKE passcode verifier. The new passcode and the discriminator
SHALL be presented to Ecosystem A Commissioner using the same rules described by ECM in
Section 5.6.3.1, “Presentation of Onboarding Payload for ECM”."

BCM is not described as possible option for JCM by the spec. So I wouldn't expect any change around OpenBasicCommissioningWindow in code.

Reply:
Consolidated all JF-related logic into jf-control-app in the latest commit. Removed OpenBasicCommissioningWindow changes as requested.